### ISSUE TITLE
Peer review

### ISSUE THREAD

**Issue by tsmith93** [STAT545-UBC-students/hw01-Caroline-Seip#1]
Hi Caroline,

Here is my feedback for your homework 1 repository: 💯 

Just kidding! I will go a bit more into detail that that. And to start off, I want to go over your README.md. Words that popped into my head after going through this file were: communicative, creative, and hilarious. The organization of this file was very clear by your use of headings. As well the use of tables and lists made this file very easy on the eyes (especially that photo of the baby caribou... well played). In general, I found you tactfully demonstrated features of Markdown files with humorous twist. Good job. One thing missing was the description of what is found in the repository.

For your R Markdown/markdown files, I thought you did an excellent job again. Not only did you run through many functions, you also went above and beyond describing exactly what information was obtained with each function. Additionally, adding the boxplot to the end of the markdown added another dimension to your database exploration. Overall, easy to follow and a diversity of functions were used.

In summary, all of the components of your homework were very well done. Clear, entertaining, and no typos 👍
